packages/react-components/src/text-input/src/PasswordInput.tsx

```diff
@@ -1,89 +1,93 @@
 import "./PasswordInput.css";
 
+import { BoxProps } from "../../box";
+import { ComponentProps, ElementType, ForwardedRef, ReactElement, SyntheticEvent } from "react";
 import { EyeIcon, PrivacyIcon } from "../../icons";
 import { IconButton } from "../../button";
-import { TextInput } from "./TextInput";
-import { bool, element, elementType, func, node, number, object, oneOf, oneOfType, string } from "prop-types";
-import { forwardRef } from "react";
-import { isNilOrEmpty, mergeProps, useControllableState, useEventCallback } from "../../shared";
+import { TextInput, TextInputProps } from "./TextInput";
+import { forwardRef, isNilOrEmpty, mergeProps, useControllableState, useEventCallback } from "../../shared";
 import { useState } from "react";
 
-const propTypes = {
+export interface InnerPasswordInputProps {
     /**
      * A controlled value.
      */
-    value: string,
+    value?: string,
     /**
      * The default value of `value` when uncontrolled.
      */
-    defaultValue: string,
+    defaultValue?: string,
     /**
      * Temporary text that occupies the input when it is empty.
      */
-    placeholder: string,
+    placeholder?: string,
     /**
      * Label identifying the input.
      */
-    label: node,
+    label?: string,
     /**
      * Whether or not a user input is required before form submission.
      */
-    required: bool,
+    required?: boolean;
     /**
      * Additional text to describe the input.
      */
-    description: string,
+    description?: string,
```

Comment:
Not sure what this prop is about, remove it please.

Reply:
done